shownotes/S01E11-白宦成：开源的魅力在于 show in public.md

```diff
@@ -11,7 +11,7 @@ hosts: '[Mikey](https://github.com/fowse2020)、[Rick](https://github.com/linuxs
 [白宦成](https://github.com/bestony)，独立开发者
 
 ## 主持人
-* [Mikey](https://github.com/fowse2020)
+* [Mikey](https://github.com/MikeyWei)，声网技术内容布道师、前技术社区媒体人，开源与安全技术爱好者。
```

Comment:
这里是不是不需要句号结尾呢，毕竟不是一个完整的句子

Reply:
Yes, we better keep the format as it was in other lines, here, it's quoted with `（）` rather than `，...。`? :), or we could change all lines' format, too.